packages/neon/neon_notifications/lib/pages/main.dart

```diff
@@ -24,33 +24,39 @@ class _NotificationsMainPageState extends State<NotificationsMainPage> {
   }
 
   @override
-  Widget build(final BuildContext context) => ResultBuilder<List<NotificationsNotification>>.behaviorSubject(
-        stream: bloc.notifications,
-        builder: (final context, final notifications) => Scaffold(
-          resizeToAvoidBottomInset: false,
-          floatingActionButton: FloatingActionButton(
-            onPressed: () async {
-              bloc.deleteAllNotifications();
-            },
-            tooltip: AppLocalizations.of(context).notificationsDismissAll,
-            child: const Icon(MdiIcons.checkAll),
-          ),
-          body: NeonListView<NotificationsNotification>(
-            scrollKey: 'notifications-notifications',
-            withFloatingActionButton: true,
-            items: notifications.data,
-            isLoading: notifications.isLoading,
-            error: notifications.error,
-            onRefresh: bloc.refresh,
-            builder: _buildNotification,
-          ),
+  Widget build(final BuildContext context) {
+    final notificationCount = bloc.notifications.value.data?.length ?? 0; // Calculate the notification count
```

Comment:
We already calculate the count in the `unreadCounter` subject, so it is better to use that.

You also need to listen on this value using `StreamBuilder` to update when the value changes. For that you should only wrap the FloatingActionButton so nothing else updates unnecessarily.